UTILS/o2dpg_report.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python3
```

Comment:
fine ... but wouldn't call this tool `o2dpg_report`. Preferably, name should tell about PR / github / ... functionality.

Reply:
Makes sense, renamed it